index.html

```diff
@@ -4286,336 +5118,508 @@ <h3 id="8_A_4">8.A.4 Example</h3>
+                                    "timezone_type": 3,
+                                    "timezone": "Europe/Amsterdam"
+                                }
+                            }
+                        ],
+                        "friday": [
+                            {
+                                "start": {
+                                    "date": "2020-02-14 09:00:00.000000",
+                                    "timezone_type": 3,
+                                    "timezone": "Europe/Amsterdam"
+                                },
+                                "end": {
+                                    "date": "2020-02-14 21:00:00.000000",
+                                    "timezone_type": 3,
+                                    "timezone": "Europe/Amsterdam"
+                                }
+                            }
+                        ],
+                        "saturday": [
+                            {
+                                "start": {
+                                    "date": "2020-02-15 09:00:00.000000",
+                                    "timezone_type": 3,
+                                    "timezone": "Europe/Amsterdam"
+                                },
+                                "end": {
+                                    "date": "2020-02-15 18:00:00.000000",
+                                    "timezone_type": 3,
+                                    "timezone": "Europe/Amsterdam"
+                                }
+                            }
+                        ],
+                        "sunday": [
+                            {
+                                "start": {
+                                    "date": "2020-02-16 12:00:00.000000",
+                                    "timezone_type": 3,
+                                    "timezone": "Europe/Amsterdam"
+                                },
+                                "end": {
+                                    "date": "2020-02-16 17:00:00.000000",
+                                    "timezone_type": 3,
+                                    "timezone": "Europe/Amsterdam"
+                                }
+                            }
+                        ]
+                    }
+                }
+            }
+        ]
+    }
 }
 </pre>
+
                 <p>
                     <strong>
-                        Get delivery options and pickup locations for Belgium
+                        Get delivery options for Belgium
                     </strong>
                     <br>
                     <br>
                     <strong>Request</strong>
                 </p>
                 <pre>
-GET https://api.myparcel.nl/delivery_options?cc=be&carrier=1&number=49&postal_code=1000  HTTP/1.1
+GET https://api.myparcel.nl/pickup_locations/?platform=myparcel&carrier=postnl&cc=be&carrier=1&number=49&postal_code=1000  HTTP/1.1

```

Comment:
waarom met een `/`?
```suggestion
GET https://api.myparcel.nl/pickup_locations?platform=myparcel&carrier=postnl&cc=be&carrier=1&number=49&postal_code=1000  HTTP/1.1

```
Zie ook andere plekken

Reply:
Done